Automattic/woocommerce-subscriptions-core
# Migrate Related Orders & Customer data store unit tests

Fixes #62

## Description


This PR migrates the Related Order and Customer data store related unit tests over to `subscriptions-core`.

Apart from moving the files there were some changes:
 - renamed the base related orders and customer data store unit test classes
 - renamed the test version of the customer and related order data stores to use "mock" instead of "test"
 - moved the test files into the same file structure as seen in `includes/`
 - updated the test filename and class names to follow the naming convention: `test-{file-name}.php` and `{Class_Name}_Test`




## How to test this PR





1. If you don't have the PHPUnit test environment set up locally, make sure to run `bin/install.sh`
2. Run `composer test`
